spikeextractors/extractors/openephysextractors/openephysextractors.py

```diff
@@ -19,7 +19,7 @@ class OpenEphysRecordingExtractor(RecordingExtractor):
     mode = 'folder'
     installation_mesg = "To use the OpenEphys extractor, install pyopenephys: \n\n pip install pyopenephys\n\n"  # error message when not installed
 
-    def __init__(self, folder_path, *, experiment_id=0, recording_id=0, dtype='float'):
+    def __init__(self, folder_path, *, experiment_id=0, dtype='int32'):
```

Comment:
can your revert this? 
An open ephys file can include multiple experiments and multiple recordings (so we need `recording_id`). I'd rather have `float` by default, so that uV are returned.

Reply:
Fixed! My sincerest apologies, I'm not sure why these changes to `openephys `were made on my end.